### PR TITLE
Password manager article

### PR DESCRIPTION
**Pull request by mfouesneau**
Added a password manager article.

### CODE REVIEW

File: faq/chapters/computing/password-managers.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Some of us may struggle to remember them, and many find tips to strengthen them, but eventually, remembering them is their primary weakness. Using multiple complex passwords can be challenging, leading many people to reuse weak passwords (sometimes with slight variations) or store them insecurely.
+
+
+## Why should we all use password managers?
+
+Worryingly, studies have shown that many people have become careless with their passwords. Nearly 35 percent of us use the same password for most online logins. Worse still, 42 percent of tech users have reported having an account hacked at least once (source [YouGov](https://today.yougov.com/technology/articles/19411-28-americans-use-one-password-most-or-all-online-l)). Despite these statistics, it is essential to understand the risks associated with poor password security and that we use our laptops and email accounts to store a massive amount of sensitive and valuable data, from banking to personal life information. People are not lax about their passwords because they don't care about their data; they are lax because they are overwhelmed by the number of passwords they need to remember.
```
Comment (ivastar) [inline faq/chapters/computing/password-managers.md:11]: "Despite" --> "Because of"?

File: faq/chapters/computing/password-managers.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+That's where password managers come in. Password managers are software applications that help you securely store and manage your passwords. They generate unique passwords for each account, store them securely, and automatically fill them in when needed. 
+
+
+Password managers offer several benefits, including:
+
+* **A password manager stores all your passwords in a single account**. The master password to your safe is the only password you’ll ever need to remember. You can also link this password to biometrics so that even that password can be highly complex. [Is it reasonable to have it all in one place?]
```
Comment (ivastar) [inline faq/chapters/computing/password-managers.md:18]: biometrics (e.g., fingerprint, facial recognition)

Comment (ivastar) [reply faq/chapters/computing/password-managers.md:18]: "[Is it reasonable to have it all in one place?]" - what do you mean here?

File: faq/chapters/computing/password-managers.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+* **Password managers generate strong, unique passwords for each of your accounts**, reducing the risk of password reuse and making it harder for hackers to gain unauthorized access.
+
+* **Password managers can automatically fill in your login credentials**, saving time and effort when logging into websites and applications.
```
Comment (ivastar) [inline faq/chapters/computing/password-managers.md:22]: Auto-fill functionality works across devices (laptop, phone, tablet) and across different applications (browsers, apps, etc.)

File: faq/chapters/computing/password-managers.md (hunk 1 above)
Comment (mfouesneau) [reply faq/chapters/computing/password-managers.md]: ✅

File: faq/chapters/computing/password-managers.md (hunk 2 above)
Comment (mfouesneau) [reply faq/chapters/computing/password-managers.md]: Should we store all passwords in a single place?

File: faq/chapters/computing/password-managers.md (hunk 3 above)
Comment (mfouesneau) [reply faq/chapters/computing/password-managers.md]: ✅

**Review by ivastar — APPROVED** [review]

**Pull request merged**